src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveClassPassTest.php

```diff
@@ -56,7 +56,7 @@ public function provideInvalidClassId()
     {
         yield [\stdClass::class];
         yield ['bar'];
-        yield [\DateTime::class];
```

Comment:
we could keep DateTime here and add DateTimeImmutable, right?

Reply:
Yes we can, but I don't think it adds any value to the test, actually my change here doesn't change any logic. I just made this stay consistent